### PR TITLE
Fixed numpy bool

### PR DESCRIPTION
**Pull request by Mehuljain12**
Hi @smtakeda , can you please review this? Thank you

### CODE REVIEW

**Comment by smtakeda** [comment]
This is for #153

File: converter.py [hunk 1]
```diff
@@ -184,9 +185,11 @@ def to_python_method(self, type_name, column):
             ctx['max_fraction'] = int(10 ** ctx['scale'])
             ctx['zero_fill'] = '0' * (9 - ctx['scale'])
         converters = [u'_{type_name}_to_python'.format(type_name=type_name)]
+        print(converters)
         if self._use_numpy:
             converters.insert(0, u'_{type_name}_numpy_to_python'.format(
                 type_name=type_name))
+            # print(converters)
```
Comment (notkriswagner) [inline converter.py]: nit: Remove this commented out code

File: test/test_numpy_binding.py [hunk 2]
```diff
@@ -2,37 +2,42 @@
 
 import numpy as np
 
-
 def test_numpy_datatype_binding(conn_cnx, db_parameters):
     """
     Tests numpy data type binding
     """
+    # print(np.bool_(True))
```
Comment (notkriswagner) [inline test/test_numpy_binding.py]: remove this too

File: test/test_numpy_binding.py [hunk 3]
```diff
@@ -2,37 +2,42 @@
...
         'specific_date': np.datetime64('1969-12-31T05:00:00')
     }, {
         'tz': 'UTC',
         'float': '-1.79769313486e+308',
         'epoch_time': epoch_time,
+        'numpy_bool': np.bool_(True),
         'current_time': np.datetime64(epoch_time, 'ns'),
         'specific_date': np.datetime64('1968-11-12T07:00:00.123')
     }]
+    print(all_data) #test
```
Comment (keller00) [inline test/test_numpy_binding.py]: More debugging prints

File: converter.py [hunk 4]
```diff
@@ -565,6 +571,9 @@ def _bytes_to_snowflake(self, value):
     def _bool_to_snowflake(self, value):
         return value
 
+    def _bool__to_snowflake(self, value): #test
+        return value
```
Comment (keller00) [inline converter.py]: I think this should be `return bool(value)` to turn it into a `bool` instead of a `np.bool_`

File: converter.py [hunk 5]
```diff
@@ -432,6 +435,9 @@ def _VARIANT_to_python(self, _):
     def _BOOLEAN_to_python(self, ctx):
         return lambda value: value in (u'1', u'TRUE')
 
+    def _BOOLEAN_numpy_to_python(self, ctx):
```
Comment (keller00) [inline converter.py]: Could you change the name of this to `_BOOLEAN_to_numpy`.

**Pull request closed without merging**